src/js/components/Button/Button.js

```diff
@@ -274,7 +275,7 @@ const Button = forwardRef(
         <Box
           direction="row"
           align="center"
-          justify={align === 'center' ? 'center' : 'between'}
+          justify={justify}
```

Comment:
For backwards compatibility maybe we could do something where if justify is not defined it defaults to `align === 'center' ? 'center' : 'between'`